Sources/SwiftSyntaxBuilderGeneration/gyb_helpers/utils.py

```diff
@@ -0,0 +1,92 @@
+from gyb_syntax_support import dedented_lines
+
+def make_swift_node(node):
+  parameters = ['name: "%s"' % node.syntax_kind]
+
+  if node.description:
+    parameters += ['description: "%s"' % flat_documentation(node.description)]
+
+  parameters += ['kind: "%s"' % node.kind]
+
+  if node.traits:
+    mapped_traits = map(lambda x: '"%s"' % x, node.traits)
+    parameters += ['traits: [%s]' % ', '.join(mapped_traits)]
+
+  if node.children:
+    children = []
+    for child in node.children:
+      children += [make_swift_child(child, 24)]
+
+    parameters += ['children: [%s]' % ',\n                  '.join(children)]
+
+  if node.collection_element != '':
+    parameters += ['element: "%s"' % node.collection_element]
+
+  if node.element_name:
+    parameters += ['elementName: "%s"' % node.element_name]
+
+  if node.collection_element_choices :
+    mapped_collection_element_choices = map(lambda x: '"%s"' % x, node.collection_element_choices )
+    parameters += ['elementChoices: [%s]' % ', '.join(mapped_collection_element_choices)]
+
+  if node.omit_when_empty:
+    parameters += ['omitWhenEmpty: true']
+
+  if node.elements_separated_by_newline:
+    parameters += ['elementsSeparatedByNewline: true']
+
+  return 'Node(%s)' % ',\n       '.join(parameters)
+
+
+def make_swift_child(child, spaces):
+  parameters = ['name: "%s"' % child.name]
+  parameters += ['kind: "%s"' % child.syntax_kind]
+
+  if child.description:
+    parameters += ['description: "%s"' % flat_documentation(child.description)]
+
+  if child.is_optional:
+    parameters += ['isOptional: true']
+
+  if child.token_choices:
+    mapped_token_choices = child.token_choices
+    if child.token:
+      mapped_token_choices.remove(child.token)
```

Comment:
I remove it here, and then add in the init, to get the exact same behaviour.

The order of the tokens matter and it will add the `syntaxKind` to the `tokenChoices` in the `Child.init` and if we not remove it here, we will have it twice in the list.

Reply:
And updated `Child.init` method